recipes/scitime/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.0.1" %}
+
+package:
+  name: scitime
+  version: {{ version }}
+
+source:
+  url: https://github.com/nathan-toubiana/scitime/archive/{{ version }}.tar.gz
+  sha256: 05f11512343bbaf521c6358e8321a8555d0ef4fe41a76ced9a8234d8b8bbab5b
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools >=3.3
+    - pip
+  run:
+    - python
+    - psutil
+    - joblib
+    - scikit-learn
+    - pandas
+
+test:
+  requires:
+    - psutil
+    - joblib
+    - scikit-learn
+    - pandas
```

Comment:
You no longer need this block of requirements since you have the run requirements which always get installed. It is best to remove it since then you don't have to maintain two lists.